lib/ansible/modules/packaging/os/yum.py

```diff
@@ -934,6 +911,35 @@ def latest(module, items, repoq, yum_basecmd, conf_file, en_repos, dis_repos, in
                 pkgs['update'].append(spec)
                 will_update.add(spec)
                 continue
+
```

Comment:
Hi I am not a maintainer, this is just opinion of "esthetics" and how comfy is to track down changes later down the line.